plugins/hello.koplugin/_meta.lua

```diff
@@ -1,6 +1,14 @@
 local _ = require("gettext")
-return {
+
+local meta = {
     name = "hello",
     fullname = _("Hello"),
     description = _([[This is a debugging plugin.]]),
 }
+if os.getenv("KODEV_ENABLE_INCOMPATIBLE_PLUGIN") then
+    meta.compatibility = {
+        min_version = "v0000.01-1",
+        max_version = "v1111.09-9",
```

Comment:
Shouldn't that be 9999?

Reply:
I'd just reduce it to KODEV_INCOMPATIBLE_PLUGIN. The "enable" is both redundant and confusing.